src/python/deepgnn/pytorch/common/consts.py

```diff
@@ -10,9 +10,6 @@
 ENCODER_TYPES = "encoder_types"
 FANOUTS = "fanouts"
 FEATURE_ENCODER_STR = "feature_encoder"
-FP16_AMP = "amp"
```

Comment:
How can we train models with half precision with ray? Where is this functionality?

Reply:
Also, const removal is a breaking change.

Reply:
This is in the run_ray function, it is just a bool now to enable or disable     train.torch.accelerate(args.fp16)